flang/include/flang/Lower/LoweringOptions.def

```diff
@@ -34,5 +34,9 @@ ENUM_LOWERINGOPT(NoPPCNativeVecElemOrder, unsigned, 1, 0)
 /// On by default.
 ENUM_LOWERINGOPT(Underscoring, unsigned, 1, 1)
 
+/// If true, add nsw flags to arithmetic operations for integer.
+/// Off by default.
+ENUM_LOWERINGOPT(NoSignedWrap, unsigned, 1, 0)
```

Comment:
`NoSignedWrap` sounds too generic for an option that only forces setting `NSW` for the loop variables' increments.  Please consider renaming or at least make the scope of `NSW` assignment clear in the comment above.

Reply:
Thank you for the review!
I changed the name of the lowering option `NoSignedWrap` to `NSWOnLoopVarInc`.